Sources/XCTest/XCTestCase.swift

```diff
@@ -48,6 +54,14 @@ private func test<T: XCTestCase>(testFunc: T -> () throws -> Void) -> XCTestCase
     }
 }
 
+// FIXME: These should be instance variables defined on XCTestCase, but when so
+//        defined Linux tests fail with "hidden symbol isn't defined". Use
+//        globals for the time being, as these seem to appease the Linux
+//        compiler.
+private var XCTLatestExpectationLocation: (file: StaticString, line: UInt) = ("", 0)
```

Comment:
I'm not sure, but I'm wondering if it may be less error-prone to not have a separate global var for this but rather to include the file+line values on `XCTestExpectation`? It doesn't feel right that this is non-optional but is set both here and below to what is effectively a placeholder value for when there is no "latest expectation".